snuba/snuba_migrations/events_analytics_platform/0001_spans.py

```diff
@@ -52,7 +52,7 @@
     Column("name", String(modifiers=Modifiers(codecs=["ZSTD(1)"]))),
     Column("sampling_factor", Float(64, modifiers=Modifiers(codecs=["ZSTD(1)"]))),
     Column("sampling_weight", Float(64, modifiers=Modifiers(codecs=["ZSTD(1)"]))),
-    Column("sign", Int(8, modifiers=Modifiers(codecs=["DoubleDelta(1)"]))),
+    Column("sign", Int(8, modifiers=Modifiers(codecs=["DoubleDelta", "ZSTD(1)"])),
```

Comment:
We don't want the `ZSTD(1)` codec for the `sign` column, we can just remove the `(1)`.

Reply:
done